### PR TITLE
Cli output misc

### PR DESCRIPTION
**Pull request by aalemayhu**
CLI output related things. Please see individual patches.

### CODE REVIEW

**Review by raybejjani — APPROVED** [review]

**Comment by aalemayhu** [comment]
Please wait with merging this until @michi-covalent has had a chance to review. Thanks.

File: pkg/endpoint/policy.go [hunk 1]
```diff
@@ -801,7 +801,7 @@ func (e *Endpoint) TriggerPolicyUpdatesLocked(owner Owner, opts models.Configura
 
 	ctCleaned = e.updateCT(owner, flushEndpointCT, consumersAdd, consumersToRm)
 
-	e.getLogger().Debugf("TriggerPolicyUpdatesLocked: changed: %d", changed)
```
Comment (aanm) [inline pkg/endpoint/policy.go:807]: `%t`

**Review by aanm — CHANGES_REQUESTED** [review]

File: pkg/endpoint/policy.go (hunk 1 above)
Comment (aalemayhu) [reply pkg/endpoint/policy.go:807]: Cool, missed that.

**Review by aanm — APPROVED** [review]

**Comment by raybejjani** [comment]
test-me-please

**Comment by aalemayhu** [comment]
test-me-please

**Review by michi-covalent — APPROVED** [review]

**Comment by ianvernon** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-all-ginkgo

**Comment by aanm** [comment]
test-all-ginkgo

**Comment by aalemayhu** [comment]
Superseded by #2587

**Pull request closed without merging**